github-orgs/kubeflow/org.yaml

```diff
@@ -933,6 +934,7 @@ orgs:
             maintainers:
             - IronPan
             members:
+            - juliusvonkohout
```

Comment:
I think adding as Kubeflow member should give you sufficient permission. Maybe removing this line?

cc @Bobgy to confirm.

Reply:
maybe @chensun knows more.

Reply:
@zijianjoy is correct. Kubeflow member is sufficient, and we should revert this line. This list seems to be exclusively for Googlers.

Reply:
Alright, I have changed it.